okhttp-brotli/api/okhttp-brotli.api

```diff
@@ -1,5 +1,8 @@
-public final class okhttp3/brotli/BrotliInterceptor : okhttp3/Interceptor {
+public final class okhttp3/brotli/BrotliInterceptor : okhttp3/CompressionInterceptor {
 	public static final field INSTANCE Lokhttp3/brotli/BrotliInterceptor;
-	public fun intercept (Lokhttp3/Interceptor$Chain;)Lokhttp3/Response;
+}
+
+public final class okhttp3/brotli/BrotliInterceptorKt {
```

Comment:
Would you consider making this a member of `BrotliInterceptor` instead? Having `Kt` in the class name is our finger in the eye of Java language users